patterns/project-description.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Title: Project Description
+ * Slug: twentytwentyfour/project-description
+ * Categories: featured
+ */
+?>
+
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"75px","right":"101px","bottom":"75px","left":"101px"}}},"backgroundColor":"custom-brown","layout":{"type":"constrained"}} -->
+<div class="wp-block-group has-custom-brown-background-color has-background" style="padding-top:75px;padding-right:101px;padding-bottom:75px;padding-left:101px"><!-- wp:columns -->
+<div class="wp-block-columns"><!-- wp:column {"style":{"spacing":{"blockGap":""}}} -->
+<div class="wp-block-column"><!-- wp:paragraph {"fontSize":"small"} -->
+<p class="has-small-font-size"><?php echo esc_html_x('Piva Hotel — Overview','Project Description pattern header','twentytwentyfour'); ?></p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"style":{"spacing":{"blockGap":""}}} -->
+<div class="wp-block-column"><!-- wp:paragraph {"fontSize":"medium"} -->
+<p class="has-medium-font-size"><?php echo esc_html_x('Sessione fotografica per il Piva Arts &amp; Crafts Hotel vicino al lago Piva, in Montenegro. Un lago artificiale è stato costruito nel 1975, dividendo il Piva Kanyon per costruire una diga Mratinje.','Project Description pattern text','twentytwentyfour');?></p>
```

Comment:
We should have only English in the copy for easier translation into other languages.

Reply:
But the design have non-english text. What should we do in that case?

Reply:
Replace it.